tests/test_app.py

```diff
@@ -172,116 +273,20 @@ def test_app_init_with_invalid_config_file(self, bad_file_name):
-                        '"projectId": "hipster-chat-mock"}',
-                        init_options=None,
-                        want_options={'databaseURL': 'https://hipster-chat.firebaseio.mock',
-                                      'projectId': 'hipster-chat-mock'}),
-        OptionsTestCase(name='partial config file is ignored',
-                        config_json='firebase_config_partial.json',
-                        init_options={'projectId': 'pid1-mock',
-                                      'storageBucket': 'sb1-mock'},
-                        want_options={'projectId': 'pid1-mock',
-                                      'storageBucket': 'sb1-mock'}),
-        OptionsTestCase(name='full config file is ignored',
-                        config_json='firebase_config.json',
-                        init_options={'databaseAuthVariableOverride': 'davy1-mock',
-                                      'databaseURL': 'https://db1-mock',
-                                      'projectId': 'pid1-mock',
-                                      'storageBucket': 'sb1-.mock'},
-                        want_options={'databaseAuthVariableOverride': 'davy1-mock',
-                                      'databaseURL': 'https://db1-mock',
-                                      'projectId': 'pid1-mock',
-                                      'storageBucket': 'sb1-.mock'}),
-        OptionsTestCase(name='full config file is ignored with missing values in options',
-                        config_json='firebase_config.json',
-                        init_options={'databaseAuthVariableOverride': 'davy1 - mock',
-                                      'projectId': 'pid1 - mock',
-                                      'storageBucket': 'sb1 - .mock'},
-                        want_options={'databaseAuthVariableOverride': 'davy1 - mock',
-                                      'projectId': 'pid1 - mock',
-                                      'storageBucket': 'sb1 - .mock'})]
-
-    @pytest.mark.parametrize('test_case', options_test_cases)
-    def test_app_init_with_default_config(self, test_case):
-        """Set the CONFIG env var and test that options are initialized"""
-        config_old = set_config_env(test_case.config_json)
-        app = firebase_admin.initialize_app(CREDENTIAL, options=test_case.init_options)
+    @pytest.mark.parametrize('env_test_case', env_options_test_cases,
+                             ids=[x.name for x in env_options_test_cases],
+                             indirect=['env_test_case'])
+    def test_app_init_with_default_config(self, env_test_case):
+        app = firebase_admin.initialize_app(CREDENTIAL, options=env_test_case.init_options)
         for field in firebase_admin._CONFIG_VALID_KEYS:
-            assert app.options.get(field) == test_case.want_options.get(field), test_case.name
-        revert_config_env(config_old)
+            assert app.options.get(field) == env_test_case.want_options.get(field)
```

Comment:
Can we do a stronger check here? Something like `assert app.options._options == env_test_cast.want_options`?